### ISSUE TITLE
A11y: Breadcrumbs text has insufficient color contrast

### ISSUE THREAD

**Issue by ThomasKindermann** [plone/volto#2489]
**Describe the bug**
The Link text of the breadcrumbs element has insufficient color contrast for a11y.
The color contrast is 3.99:1 for inactive and 4.34:1 for active links but should be above 4.5:1 for both.

**To Reproduce**
1. add two nested pages
2. go to the last page
3. open axe-dev tool and scan site
4. axe-dev tool shows color contrast violations

**Expected behavior**
axe-dev tool should show no violations for color contrast

**Screenshots**
![Screenshot 2021-06-02 at 17 32 24](https://user-images.githubusercontent.com/50659809/120509151-8e1be380-c3c8-11eb-8f63-c1643aa44a99.png)

**Software (please complete the following information):**
 - OS: osx
 - Browser chrome
 - Volto Version 13.1.2

**Comment by ThomasKindermann** [comment]
PR: https://github.com/plone/volto/pull/2491
made the text a little darker for color contrast of 4.5:1


before:
![before](https://user-images.githubusercontent.com/50659809/120775151-3ba30a00-c523-11eb-9862-bf0dfed7d2b9.png)

after:
![after](https://user-images.githubusercontent.com/50659809/120775175-41005480-c523-11eb-91cf-a1bad430bfca.png)

i changed the color in breadcrumbs.override but used a value instead of a color reference, it made the a11y test pass.
Im not sure if this is best practice, or if i should do it another way.

@sneridagh @tisto

**Comment by ichim-david** [comment]
Fix has been committed since 2021 closing this issues thus as it's solved
